hs_core/views/__init__.py

```diff
@@ -128,6 +128,25 @@ def change_quota_holder(request, shortkey):
     return HttpResponseRedirect(res.get_absolute_url())
 
 
+@api_view(['GET'])
```

Comment:
This feels more like a PUT to me because you're requesting to update the user.

Reply:
I think it's probably worth writing a tests with good and bad users.

Reply:
@hyi , how the account in Prod? HS user acc or HS admin acc or iRODS Data rods or iRODS User rods?

Reply:
@phuongdm - It'll be a regular HS user account.  The username is configured in local_settings

Reply:
don't put them on local_settings.py, it's impossible for me to read from this file on microservices, I will put them on hydroshare.re file

Reply:
@phuongdm You don't have to read local_settings.py from your hydroshare.re file, you can just put the password somewhere in your microservice configuration file on iRODS server ready to be used by hydroshare.re file. The password in local_settings.py is purely for HydroShare Django server to load for user authentication. The irods service account is just a regular user in HydroShare Django service whose credentials are shared with your iRODS microservice for user autentication before your irods microservice can update quota usage in HydroShare. I will share the password used on park with you offline for testing purpose. When it is deployed on production, you can come up with a irods service account password and then create such a user account on www.hydroshare.org, then they can communicate via API.